packages/okta-vue/package.json

```diff
@@ -54,7 +54,7 @@
     "rimraf": "^2.6.2",
     "uglifyjs-webpack-plugin": "^1.1.1",
     "url-loader": "^0.5.8",
-    "vue-loader": "^13.3.0",
+    "vue-loader": "12.2.2",
```

Comment:
Did you try 13.0.2 per https://github.com/vuejs/vue-loader/issues/1010#issuecomment-341259830 ?